pkg/virt-controller/watch/vm.go

```diff
@@ -76,6 +76,9 @@ const (
 	fetchingRunStrategyErrFmt = "Error fetching RunStrategy: %v"
 	fetchingVMKeyErrFmt       = "Error fetching vmKey: %v"
 	startingVMIFailureFmt     = "Failure while starting VMI: %v"
+
+	revisionPrefixStart    = "start"
+	revisionPrefixLastSeen = "last-seen"
```

Comment:
How do we deal with a series of rapid changes?
Does every step needs to be converted?

IIUIC then to us only the most recent state should be the relevant one?

Reply:
Yes, x rapid changes might indeed get treated as 1, or at least less than x.
The alternative was considered (and even implemented), i.e. call execute() for every single update and have it work on the generation that triggered the update (instead of the latest), but that's not a viable solution with the current state of things (the big call to `VirtualMachine().Update()` at the end would keep failing).

Reply:
Thanks, and yes, I'm also not seeing why we would worry aout the intermediate steps.

Reply:
Understood. And FWIW I don't see a need to react to every change.
We "just" need to see where we are right now, and what the user wants.